pymc_marketing/mmm/plot.py

```diff
@@ -570,7 +570,7 @@ def plot_curve(
     subplot_kwargs : dict, optional
         Addtional kwargs to while creating the fig and axes
     sample_kwargs : dict, optional
-        Kwargs for the :func:`plot_curve` function
+        Kwargs for the :func:`plot_sample` function
```

Comment:
Can we leave this change out of this PR? 🙏

Reply:
Just done this thanks